cl/donate/models.py

```diff
@@ -164,3 +166,85 @@ def __str__(self) -> str:
             self.monthly_donation_amount,
             self.get_payment_provider_display(),
         )
+
+
+class NeonWebhookEvents(models.Model):
+    MEMBERSHIP_CREATION = 1
+    MEMBERSHIP_EDIT = 2
+    MEMBERSHIP_DELETE = 3
+    MEMBERSHIP_UPDATE = 4
+    TYPES = (
+        (MEMBERSHIP_CREATION, "createMembership"),
+        (MEMBERSHIP_EDIT, "editMembership"),
+        (MEMBERSHIP_DELETE, "deleteMembership"),
+        (MEMBERSHIP_UPDATE, "updateMembership"),
+    )
+    trigger = models.PositiveSmallIntegerField(
+        help_text="The current membership tier of a user within Neon CRM",
+        choices=TYPES,
+        null=True,
+    )
+    account_id = models.CharField(
+        help_text="Unique identifier assigned by Neon CRM to a customer record",
+        default="",
+        blank=True,
+    )
+    membership_id = models.CharField(
+        help_text="Unique identifier assigned by Neon CRM to a membership record",
+        default="",
+        blank=True,
+    )
+    content = models.JSONField(  # type: ignore
+        help_text="The content of the payload of the POST request.",
+        blank=True,
+        null=True,
+    )
+
+
+@pghistory.track(AfterUpdateOrDeleteSnapshot())
+class NeonMembership(models.Model):
```

Comment:
We should include our AbstractDateTimeModel in all our models.

Reply:
got it!